diff --git a/themes/muse.zsh-theme b/themes/muse.zsh-theme
index 84bee52c..8f53b1ac 100644
--- a/themes/muse.zsh-theme
+++ b/themes/muse.zsh-theme
@@ -1,4 +1,5 @@
-PROMPT="${FG[117]}%~%{$reset_color%}\$(git_prompt_info)\$(virtualenv_prompt_info)${FG[133]}\$(git_prompt_status) ${FG[077]}ᐅ%{$reset_color%} "
+PROMPT="${FG[117]}%~%{$reset_color%}\$(git_prompt_info)\$(virtualenv_prompt_info)${FG[133]}\$(git_prompt_status)
+${FG[077]}ᐅ%{$reset_color%} "
 
 ZSH_THEME_GIT_PROMPT_PREFIX=" ${FG[012]}("
 ZSH_THEME_GIT_PROMPT_SUFFIX="${FG[012]})%{$reset_color%}"
